components/avatar/avatar.css

```diff
@@ -7,6 +7,22 @@
     
 }
 
+.text-avatar{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+    color: var(--lightgray-color);
+}
+
+.square-avatar{
+    display: flex;
+    justify-content: center;
```

Comment:
try making a different .flex class and reuse it whenever needed.

Reply:
made the changes...thanks